dagshub/data_engine/model/datasource.py

```diff
@@ -304,26 +308,44 @@ def visualize(self, **kwargs) -> "fo.Session":
     def fields(self) -> List[MetadataFieldSchema]:
         return self.source.metadata_fields
 
+    def annotate(self) -> Optional[str]:
+        """
+        Sends all datapoints in the datasource for annotation in Label Studio.
+        It's recommended to not send a huge amount of datapoints to be annotated at once, to avoid overloading
+        The Label Studio workspace.
+
+        :return: Link to open Label Studio in the browser
```

Comment:
Different docstring style from before

Reply:
This was one of the original docstring, so I didnt edit it.